apps/elf-council-frontend/src/ui/proposals/ProposalList/ProposalList.tsx

```diff
@@ -24,17 +24,24 @@ export function ProposalList({
 }: ProposalListProps): ReactElement {
   return (
     <div className="flex w-full flex-col space-y-4 pb-8">
-      {proposals.map((proposal) => (
-        <ProposalListItem
-          isGSCProposal={isGSCProposal}
-          key={proposal.proposalId}
-          active={isModalOpen && proposal.proposalId === selectedProposalId}
-          proposal={proposal}
-          onClick={onClickItem}
-          account={account}
-          signer={signer}
-        />
-      ))}
+      {proposals.map((proposal) => {
+        const activeModal =
+          isModalOpen !== undefined
+            ? isModalOpen && proposal.proposalId === selectedProposalId
+            : proposal.proposalId === selectedProposalId;
```

Comment:
this can be simplified down if you set `isModalOpen` to default `false`. We can safely assume that if `isModalOpen` is `undefined` then it's `false`